functions/go/apply-setters/README.md

```diff
@@ -9,8 +9,9 @@ Update the field values parameterized by setters.
 #### Definitions
 
 **Setters**: Setters serve as parameters for customizing field values.
-Setters are a safer option to parameterize field values as they have 
-the context of the structured data.
+Setters are a safer option to parameterize field values because, the 
```

Comment:
Grammar issue and run-on sentence, rephrase.

Reply:
No grammar issues found. But rephrased it. PTAL.